go/src/meguca/templates/thread.qtpl

```diff
@@ -121,6 +122,7 @@ Common functionality between index board pages and threads pages
 	{% code c.sticky = false %}
 	{% code c.locked = false %}
 	{% code c.rbText = config.GetBoardConfigs(c.board).RbText %}
+	{% code c.pyu = config.GetBoardConfigs(c.board).Pyu %}
```

Comment:
Please reuse the return of `config.GetBoardConfigs(c.board)`. Don't lock the mutex twice.